plugin/vite-configs/location/modals/terrain/index.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="vitest" />
+/// <reference types="vite/client" />
+
+import { defineConfig } from "vite";
+
+import { plugin } from "../../../../vite.config.template";
+
+// https://vitejs.dev/config/
+export default defineConfig(plugin("terrain"));
```

Comment:
Sorry I think this isn't necessary. I need to also remove the Datacatalog's too since they aren't used.

Reply:
done - also done for datacatalog